src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -163,7 +163,7 @@ private static bool TryGetAttributeCustomRules(Dictionary<string, OpsMetadataRul
             return attributeCustomRules.Count != 0;
         }
 
-        private static bool TryGetAllowlist(string attribute, string? listId, AllowLists allowlists, out Dictionary<string, EnumDependenciesSchema?> allowList)
+        private static bool TryGetAllowlist(string attribute, string? listId, Taxonomies allowlists, out Dictionary<string, EnumDependenciesSchema?> allowList)
```

Comment:
could you also change the method name and parameter name?

Reply:
done